etc/docs/template/static/s/js/jquery.js

```diff
@@ -0,0 +1,9789 @@
+/*!
+ * jQuery JavaScript Library v1.10.2
```

Comment:
Should we be concerned about this? If we could just try the latest jquery still works, it might be a good idea to update

Reply:
Do you mind if I make this change in my follow up PR? There's no way to test that the link I add works with the changes in this PR (since we don't actually have the whole site included).